.eslintrc.js

```diff
@@ -78,12 +78,12 @@ module.exports = {
     'prefer-destructuring': 0,
 
     /* tsx */
-    '@typescript-eslint/no-explicit-any': 'warn',
-    '@typescript-eslint/no-unused-vars': 'warn',
-    '@typescript-eslint/no-var-requires': 'warn',
-    '@typescript-eslint/no-empty-function': 'warn',
-    '@typescript-eslint/ban-types': 'warn',
-    '@typescript-eslint/ban-ts-comment': 'warn',
+    '@typescript-eslint/no-explicit-any': 0,
+    '@typescript-eslint/no-unused-vars': 0,
+    '@typescript-eslint/no-var-requires': 0,
+    '@typescript-eslint/no-empty-function': 0,
+    '@typescript-eslint/ban-types': 0,
+    '@typescript-eslint/ban-ts-comment': 0,
```

Comment:
Majority of warning are caused by `any` or empty objects and functions. And I don't know why they use `any` in codebase I think it's not accidental, it may be solves some purpose and those empty object or function represents default state of something or they used in types.
There are three ways to doing that:
- One is I can turn off them instance level.
- Another one is I can ignore them at file level.
- And last one is I can ignore them at project level.

Reply:
working on!